src/components/Datatable/ControlsModule/ColumnsControls/ColumnsControls.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react/types-6-0';
+import { action } from '@storybook/addon-actions';
+
+import ColumnsControls, { ColumnsControlsProps } from './ColumnsControls';
+import { simpleColumns } from '../../mocks/columns';
+import { useColumnsControls } from '../../hooks/useColumnsControls';
+
+export default {
+  title:
+    'components/Datatable/internalComponents/ControlsModule/ColumnsControls',
+  component: ColumnsControls,
+  parameters: {
+    viewMode: 'story',
+    docs: { disable: true },
+    previewTabs: { 'storybook/docs/panel': { hidden: true } },
+  },
+} as Meta;
+
+export const Default: Story<ColumnsControlsProps> = (args) => {
+  useColumnsControls(action('onColumnOrderChange'), simpleColumns, [
+    'country',
```

Comment:
now I see the custom order @ajkl2533 . So it's set up so that we can pass partial custom order and the rest is automatically filled. What's the reason for this preference?

Reply:
correct, this is taken from the react-table which supports this feature. Also that is the logic in the compare function. Default order is taken from the columns definition array.